scripts/normalize-yaml

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ruby
+$LOAD_PATH.unshift(File.expand_path('../lib', File.dirname(__FILE__)))
+require 'yaml_normalizer'
+require 'yaml'
+
+YamlNormalizer.run(ARGV)
```

Comment:
This script needs to be run every time we touch translations files (such as when adding new keys to make sure they're alphabetized and wrapped consitently)

To run multiple files, I run:

```
git ls-files -- config/locales | xargs ./scripts/normalize-yaml
```

I'll update our translations page in the wiki with this